### PR TITLE
Default learning rate fix

### PR DESCRIPTION
**Pull request by nandsra21**
fixes https://github.com/blab/pathogen-embed/issues/4

### CODE REVIEW

File: tests/pathogen-embed-auto.t [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Run pathogen-embed with t-SNE on a H3N2 HA alignment.
```
Comment (huddlej) [inline tests/pathogen-embed-auto.t:1]: Can you update this first line of this test and the other test in this PR to say what the test is doing more specifically? We just need a way to know that this test is checking whether `--learning-rate auto` works compared to the existing t-SNE tests which specify a learning rate value.

**Review by huddlej — COMMENTED** [review]
Thank you for adding these changes, @nandsra21. I fixed the issue with CI not running on all branches, so we don't have to specify a list of branches for it to run on.

In addition to my inline request below, can you also rename the new test files to reflect that they are running t-SNE? For example, `pathogen-embed-auto.t` should be `pathogen-embed-t-sne-auto.t`, so we know that the "auto" part refers to a t-SNE feature.

File: tests/pathogen-embed-auto.t (hunk 1 above)
Comment (nandsra21) [reply tests/pathogen-embed-auto.t]: Just pushed changes to fix this!

**Review by huddlej — APPROVED** [review]
Thanks, @nandsra21!

**Pull request merged**